.header.md

```diff
@@ -6,7 +6,7 @@ This Terraform module helps you create the base of your networking infrastructur
 
 ## Architecture
 
-![Architecture diagram](https://github.com/aws-ia/terraform-aws-network-hubandspoke/images/architecture_diagram.png)
+![Architecture diagram](https://github.com/aws-ia/terraform-aws-network-hubandspoke/blob/main/images/architecture_diagram.png)
```

Comment:
The `check vcs permalinks` lint test failed us here for some reason. Please update the link target to https://github.com/aws-ia/terraform-aws-network-hubandspoke/blob/346b078adc3fc6ace62de2ba216a9ef92666b71b/images/architecture_diagram.png to pin it to the latest commit.